content/about/contribute/github/index.md

```diff
@@ -28,30 +28,78 @@ The documentation is published under the [Apache
 
 ## How to contribute
 
-There are two ways you can contribute to the Istio documentation:
+There are three ways you can contribute to the Istio documentation:
 
 * If you want to edit an existing page, you can open up the page in your
   browser and select the **Edit This Page on GitHub** option from the gear menu
   at the top right of each page. This takes you to GitHub to edit and
   submit the changes.
 
-* If you want to work on the site in general, you must create a fork of the
-  repository. Click the button below to visit the GitHub repository. Then, you
-  must click the **Fork** button in the upper-right corner of the screen to
-  create a copy of our repository in your GitHub account. Create a clone of
-  your fork and make any changes you want. When you are ready to send those
-  changes to us, push the changes to your fork, go to the index page for your
-  fork, and click **New Pull Request** to let us know about it.
+* If you want to work on the site in general, follow the steps in our
+  [How to add content section](#add).
 
-<a class="btn btn-istio"
-href="https://github.com/istio/istio.io/">Browse this site's source
-code</a>
+* If you want to review an existing pull request (PR), follow the steps in our
+  [How to review content section](#review)
 
 Once your changes are merged, they show up immediately on
 `preliminary.istio.io`. However, the changes only
 show up on `istio.io` the next time we produce a new
 release, which happens around once a quarter.
 
+### How to add content {#add}
+
+To add content you must create a fork of the repository and a PR from
+your fork to the docs main repository. The following steps describe the
+process:
+
+1.  Click the button below to visit the GitHub repository.
+
+    <a class="btn btn-istio"
```

Comment:
This is causing a CI failure. I think the markdown process is not happy with a <a> block starting indented like that. Maybe you would just put the <a> at the end of the previous paragraph, but precede it with a <br> or two. That will likely give you the rendering you want.

Reply:
@geeknoid Thank you!

Reply:
@geeknoid Unfortunately, the fix doesn't work since the list still breaks if I do not indent the button. This is definitely a bug for the linter, where should I report it? 
I fixed it by placing the button before the list. It's not ideal but it works as a workaround.